src/Entities/Player.h

```diff
@@ -226,11 +226,14 @@ class cPlayer :
 	cWindow * GetWindow(void) { return m_CurrentWindow; }  // tolua_export
 	const cWindow * GetWindow(void) const { return m_CurrentWindow; }
 
-	// tolua_begin
-
 	/** Opens the specified window; closes the current one first using CloseWindow() */
 	void OpenWindow(cWindow * a_Window);
 
+	// tolua_begin
+
+	/** Calls OpenWindow with the pointer of it. This is exported by tolua and it adds the nil check that we want */
+	void OpenWindow(cWindow & a_Window) { OpenWindow(&a_Window); }
```

Comment:
Because of additional 10 line changes, change pointer to ref in all calls, I can change it, if you want :)